teamscale-client/src/main/java/com/teamscale/client/ITeamscaleService.java

```diff
@@ -81,33 +81,32 @@
 	);
 
 	/** Retrieve clustered impacted tests based on the given available tests and baseline timestamp. */
-	@PUT("api/projects/{projectName}/impacted-tests")
+	@PUT("api/v6.5.2/projects/{projectName}/impacted-tests?prioritization-strategy=CHEAP_ADDITIONAL_COVERAGE_PER_TIME")
```

Comment:
Is it fine that you're adding this prioritization strategy option to all calls to this endpoint, although it's only necessary for the current customer? Might we need another endpoint without this option, or might we need to parametrize this endpoint to be able to call with and without this strategy option?

Reply:
This will become the new default soon. All other options are only for research purposes, but will not be used for production systems. So I thought it would be better to keep this as an implementation detail